osu.Game/Graphics/UserInterface/ContextMenuItem.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Audio.Sample;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input;
+using osu.Game.Graphics.Sprites;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class ContextMenuItem : ClickableContainer
+    {
+        private const int height = 25;
+        private const int width = 170;
+        private const int transition_length = 200;
+
+        private Color4 backgroundHoveredColour => OsuColour.FromHex(@"172023");
+        private Color4 backgroundColour => OsuColour.FromHex(@"223034");
+
+        protected Color4 TextColour { set { text.Colour = textBold.Colour = value; } }
+
+        private readonly OsuSpriteText text;
+        private readonly OsuSpriteText textBold;
+        private readonly Box background;
+
+        public SampleChannel SampleClick, SampleHover;

```

Comment:
Private, or maybe protected if you want subclasses to be able to change the samples.